### PR TITLE
Hotfix for vm-instance to allow image names to be used correctly

### PR DESCRIPTION
**Pull request by cdunbar13**
Allows name or family to be used for the trigger, which was breaking the tests.  Exclusivity of the term is validated in variables.tf.

### CODE REVIEW

File: modules/compute/vm-instance/main.tf [hunk 1]
```diff
@@ -94,7 +94,8 @@ data "google_compute_image" "compute_image" {
 
 resource "null_resource" "image" {
   triggers = {
-    image   = var.instance_image.family,
+    name    = try(var.instance_image.name, null),
+    family  = try(var.instance_image.family, null),
     project = var.instance_image.project
```
Comment (mr0re1) [inline modules/compute/vm-instance/main.tf:99]: Let's wrap project lookup into `try` as well, for a good measure.
Please consider adding a validation test, I'm OK if you decide to do it as a separate PR.

Comment (cboneti) [reply modules/compute/vm-instance/main.tf:99]: I am not, please do add a validation test as part of this PR

**Review by mr0re1 — APPROVED** [review]

File: modules/compute/vm-instance/main.tf (hunk 1 above)
Comment (cdunbar13) [reply modules/compute/vm-instance/main.tf]: Done
